graph/skills/index.py

```
@@ -0,0 +1,268 @@
+
+    def rebuild_index(self, artifacts: list[object]) -> None:
+        """Drop all rows and re-index from *artifacts*.
+
+        Useful after schema migration or if the index becomes inconsistent.
+        """
+        conn = self._open_conn()
+        try:
+            conn.execute("DELETE FROM skills_fts")
+            conn.commit()
+        except sqlite3.Error as exc:
+            log.error("[skills] failed to clear FTS table for rebuild: %s", exc)
```

Ruff (0.15.10)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)